app/views/users/new.html.erb

```diff
@@ -0,0 +1,58 @@
+
+    <% provide(:title, "Create a new User") %>
+    <div class="container">
+        <div>
+
```

Comment:
- [ ] in `new.html.erb` we need three types of forms:
- commented out `form_tag` rails helper. :heavy_check_mark: 
- commented out html `form`. :x:
- `form_for` rails helper. :heavy_check_mark: 

Let's add the html form and make sure it works fine.